### PR TITLE
Update dependencies and use ES2015

### PR DESCRIPTION
**Pull request by fcastilloec**
Just a quick update to start getting the code up-to-date and on par with `electron-installer-debian`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,25 +31,28 @@
...
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1",
+    "mocha": "^4.0.1",
     "retry": "^0.10.1",
-    "rimraf": "^2.6.1"
+    "rimraf": "^2.6.2"
   }
```
Comment (malept) [inline package.json]: Add `engines`?

File: package.json [hunk 2]
```diff
@@ -31,25 +31,28 @@
...
-    "asar": "^0.13.0",
-    "async": "^2.1.5",
-    "debug": "^2.6.3",
-    "fs-extra": "^2.1.2",
-    "glob": "^7.1.1",
+    "asar": "^0.14.0",
+    "async": "^2.6.0",
+    "debug": "^3.1.0",
+    "fs-extra": "^4.0.3",
+    "g": "^2.0.1",
```
Comment (malept) [inline package.json]: What's this?

Comment (fcastilloec) [reply package.json]: This is very weird... I updated the modules using `ncu -a`,  this `g` module shouldn't be there. I'll make sure to remove it.

File: package.json (hunk 1 above)
Comment (fcastilloec) [reply package.json]: That's a good idea, we should do that for `electron-installer-debian` too

**Comment by fcastilloec** [comment]
> What do you think about replacing async/callbacks with Promises in a future PR? (and also in electron-installer-debian)

I like the idea of moving to Promises, I was never a fan of callbacks. Promises make code so much more readable.

**Pull request merged**